actions/python-setup-poetry/action.yaml

```diff
@@ -32,7 +32,17 @@ runs:
         python3 -m pipx ensurepath
       shell: bash
 
+    - name: Cache poetry
+      id: cache-poetry
+      uses: actions/cache@v4
+      with:
+        path: |
+          /opt/pipx/venvs/poetry
+          /opt/pipx_bin/poetry
+        key: poetry-${{ inputs.poetry-version }}-${{ runner.os }}
```

Comment:
We should probably include `python-version` in the cache key

Reply:
done